wherehows-common/src/test/java/wherehows/common/utils/JobsUtilTest.java

```diff
@@ -116,6 +101,25 @@ public void testGetEnabledJobsByType() throws IOException {
     Files.deleteIfExists(path2);
   }
 
+  @Test
+  public void testMultipleLinesSameProperties() throws IOException, ConfigurationException {
+    String dir = "/tmp/";
+    String propertyStr =
+        "var1=foo\n" + "var1=foo\n" + "var1=foo\n" + "var2=foo\n" + "var2=foo\n" + "var3=foo\n" + "var1=foo\n"
+            + "var4=foo\n";
+    Properties props = new Properties();
+    Path path = createPropertiesFile(propertyStr);
+
+    props = getJobConfigProperties(path.toFile());
+
+    Assert.assertNotEquals(props, null);
+    Assert.assertEquals(props.getProperty("var1", ""), "foo,foo,foo,foo");
+    Assert.assertTrue(props.getProperty("var2").length() > 0);
```

Comment:
Let's do simple literal value comparison for var2, var3, & var4.

Reply:
Did literal value comparison now.